server/src/projects/projects.service.ts

```diff
@@ -46,6 +45,47 @@ export class ProjectsService {
     }
   }
 
+  /**
+   * Gets a Project for given dcp_projectid and contactid
+   *
+   * @param      {string}  projectId   CRM Project dcp_projectid
+   * @param      {string}  contactId   CRM Contact contactid
+   * @return     {any}     { ...Project Attributes }
+   */
+  public async getProject(projectId: string, contactId: string) {
+    try {
+      const { records } = await this.crmService.get('dcp_projects', `
+        $filter=
+          dcp_dcp_project_dcp_projectapplicant_Project/
+            any(o:
+              o/_dcp_applicant_customer_value eq '${contactId}'
```

Comment:
This looks good but I'm still not sure why we check contactId here? Is this an authorization step?

I'm fine with this for now, but we need to come up with a broader authorization strategy. It's weird that projects authorize but not packages, etc.

Reply:
Yep, for auth. Yeah, agreed we need a broader strategy. We had planned that during the next sprint (or sooner) we will make sure packages are locked down. I just wanted to be proactive here about locking it down, since it was an easy step to take here

https://app.zenhub.com/workspaces/labs-sprint-5e56ebd113bedb2575e694da/issues/nycplanning/labs-applicant-portal/507